benchmark/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ target_include_directories(${APP_TARGET}
 
 target_sources(${APP_TARGET}
     PRIVATE
-        main.cpp
```

Comment:
Please delete main.cpp because it's not used anymore and Mbed CLI 1 can pick it up